fs.go

```diff
@@ -1130,7 +1147,7 @@ const (
 	fsMaxCompressibleFileSize = 8 * 1024 * 1024
 )
 
-func (h *fsHandler) compressAndOpenFSFile(filePath string, fileEncoding string) (*fsFile, error) {
+func (h *fsHandler) compressAndOpenFSFile(path, filePath string, fileEncoding string) (*fsFile, error) {
```

Comment:
Can you come up wit ha better name for `path` here?

Reply:
Which do you think better?
- pathStr https://github.com/valyala/fasthttp/blob/b23c5e90838fdcb504032d8932c905ca7bd3b7ae/fs.go#L833
- pathOrig
- uri